app/models/alchemy/page/page_elements.rb

```diff
@@ -199,7 +199,7 @@ def element_definitions_by_name(names)
     #     feed_elements: [element_name, element_2_name]
     #
     def feed_elements
-      elements.named(definition['feed_elements'])
+      elements.published.named(definition['feed_elements'])
```

Comment:
Yes. That's a good point. I was worried that I exposed trashed elements in our application by only using the `published` scope. But thankfully [trashing the element will also unpublish it]( https://github.com/AlchemyCMS/alchemy_cms/blob/3c2753ff27b2a602a1209390678e90f8ec258707/app/models/alchemy/element.rb#L233).